### ISSUE TITLE
Dappstore is not clear enough

### ISSUE THREAD

**Issue by pablomendezroyo** [dappnode/DNP_DAPPMANAGER#519]
**Is your feature request related to a problem? Please describe.**
There are a few users that do not detect the DAppStore as clear as should be. Foe example  PRYSM  looks more like an ad and not like a DAPP. This may confuse users.

**Describe the solution you'd like**
A solution may be to just add an get/update file inside those cards that do not have this feature (this only happens in new cards for the Prysm and DMS packages).

**Describe alternatives you've considered**

**Additional context**
User do not detect Prysm package as a clickable card.

![Screenshot from 2020-12-18 08-45-11](https://user-images.githubusercontent.com/41727368/102588477-c88d0f00-410d-11eb-8f56-0e882d23fc43.png)

**Comment by dapplion** [comment]
What do you think about showing the featured packages in both places, in the main featured to banner and as a regular dapp card?

**Comment by pablomendezroyo** [comment]
> What do you think about showing the featured packages in both places, in the main featured to banner and as a regular dapp card?

Yeah, we could also consider having an adds section where new packages or new releases will arrived.

**Comment by olofedlund** [comment]
It would be helpful if the color of the button for “Update” was more different in color from the Get Green. It is hard to get a quick overview in the store for what can be updagted.

**Comment by olofedlund** [comment]
Could the sort order in the store be changed? First packages that needs to be updated, then installed, then packages in status “Get”.

**Comment by pablomendezroyo** [comment]
This issue must be splited into concrete issues

**Comment by alexpeterson91** [comment]
> This issue must be split into concrete issues

One should be organizing the dappstore, just putting headings on each category, i.e. Gnosis chain, ETH Mainnet, ETH Göerli/Prater, Gnosis Chiado, metrics packages, other blockchains (bitcoin, avalanche, ETC, Monero, Pokt, Hopr) and so on.  Was thinking about this a day or 2 ago.

**Comment by pablomendezroyo** [comment]
Wont implement
